cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -48,6 +48,7 @@ spec:
         args:
         # command = "/kube2sky"
         - -domain={{ pillar['dns_domain'] }}
+        - -kube_master_url=http://10.0.0.1:8080
```

Comment:
I'm assuming that 10.0/16 is the portal_net for you ?
We would have to get this parameter from the "pillar".